docs/assets/javascripts/version-select.js

```diff
@@ -3,17 +3,12 @@
     var VERSION = window.location.pathname.split("/")[1];
     var VERSION_LATEST = ".";
 
-    function removePrefix(str, prefix) {
-        var hasPrefix = str.indexOf(prefix) === 0;
-        return hasPrefix ? str.substr(prefix.length) : str.toString();
-    }
-
     function populateVersionSitemap(version) {
         var versionPath = version === VERSION_LATEST ? "" : "/" + version;
         window.versionPages[version] = [];
 
         var xhrSitemap = new XMLHttpRequest();
-        var sitemapURL = window.location.origin + versionPath + "/sitemap.xml";
+        var sitemapURL = window.location.origin + versionPath + '/sitemap.xml';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/codacy/docs/pullRequest?prid=26905)
```suggestion
        var sitemapURL = window.location.origin + versionPath + "/sitemap.xml";
```